scripts/data_guide/data_guide.qmd

```diff
@@ -1,4 +1,4 @@
----
+ce---
```

Comment:
Why the ce?

Reply:
This was accidental. I corrected it in the latest commit. Thanks for catching it!